lib/Doctrine/ORM/Internal/Hydration/AbstractHydrator.php

```diff
@@ -17,11 +17,13 @@
 use Doctrine\ORM\Tools\Pagination\LimitSubqueryWalker;
 use Doctrine\ORM\UnitOfWork;
 use Generator;
+use LogicException;
 use ReflectionClass;
 use TypeError;
 
 use function array_map;
 use function array_merge;
+use function assert;
```

Comment:
This is never used, is it?

Reply:
Oh right, it's a remnant from a first attempt at fixing this, I'll remove it 👍